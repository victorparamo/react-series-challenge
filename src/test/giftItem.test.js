import React from 'react';
import { shallow } from 'enzyme';
import GiftItem from '../components/giftItem.js';

describe('test', () => {
  it('will work', () => {
    const component = shallow(<GiftItem 
                                    url={"https://media3.giphy.com/media/3ktRws88PRbBVbbYyE/giphy-preview.gif"}
                                    name={"item.title"}
                                    id={"item.id"}
                                    key={1}/>);
    expect(component).toMatchSnapshot();
  });
});
