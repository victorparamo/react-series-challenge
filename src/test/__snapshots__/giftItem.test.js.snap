// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test will work 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <GiftItem
    id="item.id"
    name="item.title"
    url="https://media3.giphy.com/media/3ktRws88PRbBVbbYyE/giphy-preview.gif"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "alt": "",
      "onClick": [Function],
      "src": "https://media3.giphy.com/media/3ktRws88PRbBVbbYyE/giphy-preview.gif",
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "alt": "",
        "onClick": [Function],
        "src": "https://media3.giphy.com/media/3ktRws88PRbBVbbYyE/giphy-preview.gif",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
